scss/styles.scss

```diff
@@ -17,4 +17,5 @@ $margin-big: 15px;
 
 body {
     font-family: 'Roboto', sans-serif;
+    padding-top: 90px;
```

Comment:
its a margin, not a padding. You should use the nav-height variable in case the navbar changes its height in a future